mixer/tools/codegen/cmd/mixgenbootstrap/main.go

```diff
@@ -61,6 +61,9 @@ func withArgs(args []string, errorf func(format string, a ...interface{})) {
 			importMapping := make(map[string]string)
 			for _, maps := range mappings {
 				m := strings.Split(maps, ":")
+				if len(m) != 2 {
+					errorf("Invalid map %s, map should only contain one colon", maps)
```

Comment:
@zhuguihua I am just curious, how did you run into this issue? One would run into this when specifying adding additional import mapping. AFAIU, this tool is mainly used by a very specific script bin/mixer_codegen.sh which already contains some import mapping, and I personally have not seen users needing additional import mapping beyond what the script already has.

Reply:
@guptasu  Sorry, I did not actually use this script, I was just learning this piece of code, feel that if not checked, this may encounter the problem of an empty array.

Reply:
:+1: good catch. Makes total sense to fix it. I was just curious how you caught the issue.